### PR TITLE
Update category table for libraries on NPM

### PR DESCRIPTION
**Pull request by Fryuni**
#### Description

- Rename `collection` to `category` to match the name in the [integrations library page](https://astro.build/integrations)
- Add "Dev Toolbar" category:
	- Added as "Developer Tools" in https://github.com/withastro/astro.build/pull/878
	- Renamed to "Dev Toolbar" in https://github.com/withastro/astro.build/pull/906
- Add "Loader" and "Utilities" categories:
	- Added in https://github.com/withastro/astro.build/pull/1281
- Document default "Uncategorized" category:
	- Added in https://github.com/withastro/astro.build/pull/1281

### CODE REVIEW

**Review by sarah11918 — APPROVED** [review]
Fantastic! Will also note, we added `loader` as a "category" in the beta branch, I believe. BUT there's no reason that can't be done right now, too! Even if Loader API is experimental, people can still build loaders. Would you be willing to add that category on astro.build too?

**Comment by sarah11918** [comment]
Also, would it make sense to make an "other" category for integrations to default to? (Non-blocking!)

**Comment by Fryuni** [comment]
Sure thing, do you think it can be done in the [same PR](https://github.com/withastro/astro.build/pull/1281) or have one for each (utilities, loader and other)?

File: src/content/docs/en/reference/publish-to-npm.mdx [hunk 1]
```diff
@@ -279,18 +279,22 @@ Avatars are a great way to highlight your brand in the library! Once your packag
...
 
-| collection        | keywords                                     |
+| category          | keywords                                     |
 |------------------ | -------------------------------------------- |
 | Accessibility     | `a11y`, `accessibility`                      |
 | Adapters          | `astro-adapter`                              |
 | Analytics         | `analytics`                                  |
 | CSS + UI          | `css`, `ui`, `icon`, `icons`, `renderer`     |
 | Frameworks        | `renderer`                                   |
 | Performance + SEO | `performance`, `perf`, `seo`, `optimization` |
```
Comment (sarah11918) [inline src/content/docs/en/reference/publish-to-npm.mdx:293]:
```suggestion
| Content Loaders      | `astro-loader`                                                    |
| Performance + SEO | `performance`, `perf`, `seo`, `optimization` |
```
Here's what we already drafted in the beta branch!

**Comment by sarah11918** [comment]
> the name in the [integrations library page](https://astro.build/integrations)

Will also point out that we do not have an agreed upon, consistent name for the integrations library/catalogue/directory 😅

**Review by apatel369 — APPROVED** [review]

**Review by delucis — APPROVED** [review]
Thank you @Fryuni!

**Pull request merged**